sdk/azcore/cloud/cloud.go

```diff
@@ -1,6 +1,3 @@
-//go:build go1.18
```

Comment:
There are more cases of this, surprised they didn't get flagged on my local runs.

Reply:
Env issue.